bgs/bgs.go

```diff
@@ -198,7 +198,8 @@ func (bgs *BGS) Start(listen string) error {
 	e.GET("/xrpc/_health", bgs.HandleHealthCheck)
 
 	admin := e.Group("/admin", bgs.checkAdminAuth)
-	admin.POST("/deleteRecord", bgs.handleAdminDeleteRecord)
+	admin.POST("/subs/setEnabled", bgs.handleAdminSetSubsEnabled)
+	admin.POST("/repo/takedown", bgs.handleAdminTakedownRepo)
```

Comment:
I think we need "un-takedown", even from the get-go. The lesson from PDS is that even if we only think we will use this in extreme cases, if it is a lever then in an emergency we might use it, and need a way to undo it.

Fortunately, that handler should be quite simple: just flip the boolean in the database table.

Reply:
I dont know about that. We shouldnt ever do a takedown at the bgs level lightly. If we make that decision, we should have zero plans to un-make it.

Reply:
added the route anyways

Reply:
thanks!